heat/core/logical.py

```diff
@@ -214,6 +180,47 @@ def local_any(t, *args, **kwargs):
     )
 
 
+def isclose(x, y, rtol=1e-05, atol=1e-08, equal_nan=False):
+    """
+    Parameters:
+    -----------
+
```

Comment:
remove extra blank line between  '--------' and text

Reply:
Done